Jamfile

```diff
@@ -56,15 +56,52 @@ rule linking ( properties * )
 {
 	local result ;
 
+	# ssl=openssl => crypto=libcrypto
+	# this match the previous behavior
+	if <ssl>openssl in $(properties)
+		&& ! <crypto>libcrypto in $(properties)
+	{
+		EXIT "with ssl=openssl the value for crypto must be libcrypto" ;
```

Comment:
What can I say, I code in bjam like I'm learning an alien language, let me research this